build.gradle

```diff
@@ -4,9 +4,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0-beta5'
+        classpath 'com.android.tools.build:gradle:2.1.0-alpha4'
```

Comment:
Are you using something from the latest alpha release? If not, I'd prefer to leave this dependent on the theoretically more stable beta* releases if we can't use the full releases.